src/main/java/com/apigee/edge/config/rest/RestUtil.java

```diff
@@ -477,5 +697,31 @@ public static HttpResponse getAPIConfig(ServerProfile profile,
         return response;
     }
 
+    public static HttpResponse deleteAPIResourceFileConfig(ServerProfile profile, String api, String resource, String resourceId)
+			throws IOException {
+
+		String importCmd = profile.getHostUrl() + "/" + profile.getApi_version() + "/organizations/" + profile.getOrg()
+				+ "/apis/"+api
+				+ "/" + resource + "/" + resourceId;
```

Comment:
Is URLEncoder.encode the only difference between this and deleteAPIConfig?

Reply:
I believe - yes, should I just add another argument to the method to say if urlencode and then use that to build the URL ??
Something like this
```java

public static HttpResponse deleteAPIResourceFileConfig(ServerProfile profile, String api, String resource, String resourceId, boolean encode){
  return (deleteAPIConfig(profile, api, resource, resourceId, false));
}

public static HttpResponse deleteAPIConfig(ServerProfile profile, String api, String resource, String resourceId, boolean encode){
  if(encode)
   String importCmd = profile.getHostUrl() + "/"
                            + profile.getApi_version() + "/organizations/"
                            + profile.getOrg() + "/apis/"
                            + api + "/" + resource + "/"
                            + URLEncoder.encode(resourceId, "UTF-8");
  else
    String importCmd = profile.getHostUrl() + "/" 
                                      + profile.getApi_version() 
                                      + "/organizations/" 
                                      + profile.getOrg()
                                      + "/apis/"+api+ "/" + resource 
                                      + "/" + resourceId;
......
}
```